### PR TITLE
Fix some clang-tidy issues on Travis

### PR DESCRIPTION
**Pull request by rhaschke**
Having switched to Noetic (and clang-tidy-10?) on Travis, we get some new clang-tidy issues reported:
https://travis-ci.com/github/ros-planning/moveit/jobs/393573194

Interestingly, locally I got even more issues!? There is also one "fix" applied that is nonsense:
```diff
-struct DistanceResultsData
+struct i0
```

### CODE REVIEW

**Comment by tylerjw** [comment]
I've seen this too and I don't know what to do with it.

**Review by tylerjw — APPROVED** [review]
All the typedef to using changes look good.  It is just a new style of aliases.  I remember reading Scott Myers where he advises using the new style.

**Comment by rhaschke** [comment]
Forgot to clang-format what was fixed by clang-tidy... Should be fine now.

**Comment by v4hn** [comment]
> It is just a new style of aliases.

Goodbye typedef! Using using is the only proper way to define aliases by now and it annoyed me for some time already that MoveIt is still full of typedefs...

**Comment by rhaschke** [comment]
Thanks to clang-tidy this was _automatically_ fixed now :wink:

File: moveit_core/collision_detection/include/moveit/collision_detection/collision_common.h [hunk 1]
```diff
@@ -356,7 +356,7 @@ struct DistanceResultsData
 };
 
 /** \brief Mapping between the names of the collision objects and the DistanceResultData. */
-typedef std::map<const std::pair<std::string, std::string>, std::vector<DistanceResultsData> > DistanceMap;
+using DistanceMap = std::map<const std::pair<std::string, std::string>, std::vector<DistanceRequest> >;
```
[the remark this answers is not in the document]
Comment (tylerjw) [reply moveit_core/collision_detection/include/moveit/collision_detection/collision_common.h]: very weird

**Pull request merged**